config/test.json

```diff
@@ -39,6 +39,7 @@
 		"ADMIN_USER": "dbildungscloud",
 		"ADMIN_PASSWORD": "dBildungscloud"
 	},
+	"LOG_LEVEL" : "emerg",
```

Comment:
Den Wert würde ich hier nicht ändern.

Reply:
it has not been changed, was emerg before for test, might be because of feathers tests and their log output?